data_registry/templates/includes/files.html

```diff
@@ -0,0 +1,27 @@
+{% load i18n %}
+{% load registry %}
+
+<ul>
+{% with files=files|get_item:key %}
+    {% if files.full %}
+    <li>
+        <a href="{% url 'download_export' %}?spider={{ collection.source_id }}&job_id={{ job.id }}&full=true&suffix={{ suffix }}" rel="nofollow" download>
```

Comment:
`collection.source_id` and `job.id` are not seen from the template.

Reply:
Aha, need to add to templatetag